build/elasticsearch/bin/docker-entrypoint.sh

```diff
@@ -59,4 +93,11 @@ if bin/elasticsearch-plugin list -s | grep -q x-pack; then
     fi
 fi
 
-exec bin/elasticsearch "${es_opts[@]}"
+if [[ "$(id -u)" == "0" ]]; then
+    # If requested and running as root, mutate the ownership of bind-mounts
+    if [[ -n "$MUTATE_MOUNTS" ]]; then
```

Comment:
`$TAKE_FILE_OWNERSHIP`? It's not obvious what kind of "mutation" is going to happen.

Reply:
Addressed in 74ca02ab848160ea57904307d1ed7594fbc085d6